### PR TITLE
Adding UI for quiz creation

### PR DESCRIPTION
**Pull request by megsachdev**
## PR Checklist: Pending

- [ ] The commit message follows our guidelines: https://github.com/BioMediaLab/whiteboard/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

- [X ] Feature --> Added the UI for the quiz creation, the data flow is yet to be handled
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

This is the page to handle quiz creation, though the data flow is yet to be handled

Issue Number: #10 

## What is the new behavior? 
This is the UI for quiz creation.
Screenshot from 2019-05-17 10-01-07

## Does this PR introduce a breaking change?

- [ ] Yes
- [ X] No

### CODE REVIEW

File: src/graphql/schema.json [hunk 1]
```diff
@@ -2735,6 +2735,23 @@
         "onOperation" : false,
         "onFragment" : false,
         "onField" : true
+      }, {
```
Comment (cliffpyles) [inline src/graphql/schema.json]: Is it possible this change was made when the initial dev environment was being setup?

File: src/pages/classes/_classId/quizzes/new.js [hunk 2]
```diff
@@ -20,15 +21,18 @@ const reducer = (current, action) => {
...
     quizClassId: classId,
     title,
+    description,
     questions: []
   }
+  const [questions, setState] = useState([]);
   const [state, dispatch] = useReducer(reducer, initialState)
   const handleSubmit = event => {
-    mutation({ input: state })
+    console.log(questions);
```
Comment (cliffpyles) [inline src/pages/classes/_classId/quizzes/new.js]: Looks like there might be a few console.log statements left. Would you mind removing those? I'll try to get some automation setup that will help warn about this in the future.

File: cypress/utils/sign-in.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function signIn() {
+    cy.get('input[name=username]').type('megha');
+    cy.get('input[name=password]').type('June@120688');
```
[the remark this answers is not in the document]
Comment (megsachdev) [reply cypress/utils/sign-in.js]: I moved it to the environment variables. We can move it to Travis once we set it.

File: src/pages/classes/_classId/quizzes/new.js (hunk 2 above)
Comment (megsachdev) [reply src/pages/classes/_classId/quizzes/new.js]: Done

File: src/components/Checkbox/Checkbox.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { Checkbox } from '@shopify/polaris'
+
```
Comment (andyk2177) [inline src/components/Checkbox/Checkbox.js]: Is there a reason why we are just re-exporting `polaris` components in our components folder?

**Comment by andyk2177** [comment]
@megsachdev has confirmed this was a WIP PR that is no longer needed so closing.

**Pull request closed without merging**